server/handlers.go

```diff
@@ -233,6 +233,15 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if authReq.ConnectorID != "" {
+		for _, c := range connectors {
+			if c.ID == authReq.ConnectorID {
```

Comment:
❓ Should this return an error if `connector_id` was specified but doesn't match any?

Reply:
I debated this myself.  What could it ever do with an error other than break flow completely or just return to the index?

Reply:
That's true, but it's still better than 🙈ignoring it, isn't it? I'd propose calling `s.tokenErrHelper`, maybe?

Reply:
Perhaps if `connector_id` does not match any, then all connectors are shown as they were prior to this PR?

Reply:
Gotcha.

Reply:
This would not change anything with respect to showBacklink (#1123), since the number of connectors doesn't actually change.  It would need an additional test such as authReq.ConnectorID != "" to suppress the link.

Reply:
I'm not sure we'd actually _want_ to suppress the link, would we? Maybe you'd like to use a different method...? 🤔